src/CommunityToolkit.Maui/Mappings/SizeMappingInfo.shared.cs

```diff
@@ -0,0 +1,41 @@
+﻿using CommunityToolkit.Maui.Core.Primitives;
+
+namespace  CommunityToolkit.Maui.Mappings;
+
+/// <summary>
+/// Size mapping info about the screen diagonal-size and the category it falls into.
+/// </summary>
+public class SizeMappingInfo
+{
+	/// <summary>
+	/// 
```

Comment:
need to provide info
```suggestion
	/// Initializes an instance of <see cref="SizeMappingInfo"/>.
```